db/portfolio_2018-10-03.sql

```diff
@@ -0,0 +1,82 @@
+# ************************************************************
```

Comment:
why do you have 4 versions of your DB in the repo? which one am i supposed to use?